config.go

```diff
@@ -20,8 +20,6 @@ const (
 
 var disallowedPortsBitset [MAX_DISALLOWED_PORT_NUM + 1]bool
 
-var disallowedPortsBitsetInitialized = false
-
 func initDisallowedPortsBitset() {
```

Comment:
It looks to me like maybe `initDisallowedPortsBitset` should set `disallowedPortsBitsetInitialized = true`?